### PR TITLE
Fix issue with persisting after refresh

### PR DESCRIPTION
**Pull request by nalanart**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -12,18 +12,13 @@ import { Redirect } from "react-router-dom";
 
 function App() {
   const { isAuthenticated } = useContext(UserContext);
+  // console.log(isAuthenticated);
```
Comment (WhoaNellie) [inline client/src/App.js]: Can remove commented out code/console logs

File: client/src/pages/HomePage.js [hunk 2]
```diff
@@ -14,41 +17,21 @@ const useStyles = makeStyles((theme) => ({
...
-      image: "https://picsum.photos/400",
-      content:
-        "Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisUt enim ad minim veniam, quis nostrud exercitation ullamco laboris nisquis nostrud exercitation ullamco laboris nisUt eni",
-    },
-  ];
+  const { searchTerm, loading, isAuthenticated } = useContext(UserContext);
+
+  // useEffect(() => {
+  //   axios.get(`/api/mentions?keyword=${searchTerm}`)
+  // }, [searchTerm])
```
Comment (WhoaNellie) [inline client/src/pages/HomePage.js]: Ditto here

File: client/src/pages/HomePage.js [hunk 3]
```diff
@@ -1,8 +1,11 @@
-import React from "react";
+import React, { useContext, useEffect } from "react";
```
Comment (WhoaNellie) [inline client/src/pages/HomePage.js]: Can remove useEffect if you're not using it here

**Review by WhoaNellie — COMMENTED** [review]
Just left a few comments about cleanup.
Never seen useEffect inside a context provider, but if it works I think that it's a good idea!

**Comment by nalanart** [comment]
Cleaned up the console logs !

**Review by jessdesr — COMMENTED** [review]
So ideally, what you'll want to trigger a call to fetch the user profile whenever the user refreshes the page.  By default isAuthenticated should always be false, and upon receiving a response from the user profile endpoint, it should then be updated to be true.

**Comment by nalanart** [comment]
New branch was created to address these issues

**Pull request closed without merging**